applications/skyepass.md

```diff
@@ -0,0 +1,251 @@
+
+Some ideas we have had so far:
+
+- `Crypto Wallet`: shared hot wallet. The owner of the vault can install an `Ethereum` extension and store the private key with it. And, of course, DApp browsers. 
+- `SSH Login Tool`: a whole team can share login credential to their server effortlessly. 
+- `Shared Phone Number`: a shared Google account that registered on `Google Voice` can be stored, and the whole family can receive verification code for services. 
+
+
+
+### Password Manager & an Identity Management Solution
+
+Based on some thinking of the basis nature of NFTs. We believe that a password manager is an ideal medium to deliver tokenized digital identities. Therefore, we think each username-password-OTP combination as an atomic token, a vault as a collection of these identities, and an `extension` as a service injected with an identity. 
+
+- If we assume all identity tokens have two states: "public identity" or "private identity". A public handle is the public identity of a user. (i.e. a twitter handler, a Github handle or a Venmo handle etc. ) Therefore, we are building a solution to link to one's public off-chain profiles. Also, we can implement a ENS-like or `@username` style handle system.
+- Therefore, simple sharing behavior (i.e. share my spotify account to my girlfriend) can take two forms: if she has an account with this password manager, simple `@her`, set some rules for using this password(or not) and press share. If she has not, a one time sharing link will be sent, her browser will generate an ephemeral key pair, and that ephemeral key pair will be used to encrypted the entry and send the encrypted password entry over and make it self-destruct soon.
+- For teams or families, they are using a shared identity. They can link their profiles and get a handle like `@team`, while the team will use some secret sharing schema for privilege management.
+
+For more on this, please refer to `Future Plan/Integrated Identity Solution` section. 
+
+### Ecosystem Fit 
+I don't think there are anything like SkyePass so far, both within the Substrate community or all blockchain communities. The reason that Substrate will be an ideal platform for SkyePass is because the flexibility the framework offers. We plan to deploy 4 smart contracts:
+
+- Contract 1: A NFT contract that issues an NFT token for vault creator, store metadata of the vault and manage permission to update the vault. (the `vault metadata contract`)
+
+- Contract 2: A NFT contract that generate an NFT token that represents an atomic digital identity to users (the `atomic digital identity contract`)
```

Comment:
What is this needed for?

Reply:
Contract2? Explained on your earlier question on password sharing.